openquake/hazardlib/calc/filters.py

```diff
@@ -126,6 +126,12 @@ def get_distances(rupture, sites, param, dcache=None):
     elif param == 'closest_point':
         t = rupture.surface.get_closest_points(sites)
         dist = numpy.vstack([t.lons, t.lats, t.depths]).T  # shape (N, 3)
+    elif param == 'clon' or param == 'clat':
+        t = rupture.surface.get_closest_points(sites)
```

Comment:
`rupture.surface.get_closest_points(sites)` will be called twice anyway, so this change produced not speedup at all. Please at a comment

# TODO: measure the performance penalty caused by the double call to `.get_closest_points`
# it could very well be insignificant and in that case it is not worth doing more

Reply:
I have added this comment.